tests/unit/tasks/test_generate_queries.py

```diff
@@ -622,9 +662,17 @@ def test_it_throws_for_unconvertable_matches(self):
                 },
             )
 
-    def test_it_throws_for_invalid_schema(self):
-        with pytest.raises(ValueError):
+    def test_it_throws_for_invalid_schema_for_inner_children(self):
+        with pytest.raises(ValueError) as e:
             get_inner_children("struct<name:string", "struct<", ">")
+        assert e.value.args[0] == "Column schema is not valid"
+
+    def test_it_throws_for_invalid_schema_for_nested_children(self):
+        with pytest.raises(ValueError) as e:
+            get_nested_children(
+                "struct<name:string,age:int,s:struct<n:int>,b:string", "struct"
+            )
+        assert e.value.args[0] == "Column schema is not valid"
```

Comment:
This is slightly unrelated, there was no coverage for it so I just added those to make sure that scenario is handled